gems/sorbet-runtime/lib/types/private/methods/signature.rb

```diff
@@ -54,8 +50,6 @@ def initialize(method:, method_name:, raw_arg_types:, raw_return_type:, bind:, m
     @parameters = parameters
     @on_failure = on_failure
     @override_allow_incompatible = override_allow_incompatible
-    @generated = generated
-    @ever_failed = false
```

Comment:
Why did `@ever_failed` disappear too?

Reply:
The only place that we were using it was to say, "if a generated sig fails, put back the original sig (at least until the next deploy / service restart)" as a sort of circuit breaker. That way, in the case that there's a huge problem with a generated sig, it won't spiral out of control (e.g., spend tons of time logging / erroring / etc.).

Since it's sole purpose was for `generated` sigs, we don't need it anymore.